corehq/apps/app_manager/templates/app_manager/v1/partials/shadow_parent_select.html

```diff
@@ -1,10 +1,21 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Title</title>
-</head>
-<body>
+{% load i18n %}
+{% load hq_shared_tags %}
 
-</body>
-</html>
+<div class="form-group" xmlns="http://www.w3.org/1999/html">
+    <label class="control-label col-sm-2">
+        {% trans "Shadow parent" %}
```

Comment:
I was using "parent" to avoid the "source" collision, but I think for the UI it's better to say "source".